src/Symfony/Component/HttpKernel/RebootableInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel;
+
+/**
+ * Rebotable extends the Kernel to allow it to reboot using a temporary cache directory.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface RebootableInterface
```

Comment:
By the method is used only in the cache:clear command, not sure that accounts for "a lot" :)